.github/workflows/build.yml

```diff
@@ -0,0 +1,20 @@
+name: android-build
```

Comment:
I recommend explicitly quoting all yaml strings.  eg:

```
name: "android-build"
```

and so on throughout the rest of this and the other .yml file.  This avoids surprising misunderstandings when you happen to have a string value like "true" or "yes" (or any of the many other non-string literals yaml supports).